docs/client/AndroidOnDeviceEvaluation/_globalUser.mdx

```diff
@@ -0,0 +1,11 @@
+### Setting a Global User
+
+To avoid needing to pass the user object to every single evaluation call, you can set a global user.
+This user will be used for all evaluations unless otherwise specified.
```

Comment:
Then, when checking a gate/experiment/layer, provide "null" for the user to use the global user